packages/lit-dev-content/src/components/litdev-search.ts

```diff
@@ -125,14 +129,14 @@ class LitDevSearch extends LitElement {
       }
 
       :host {
-        padding-left: 0;
+        padding-inline-start: 0;
```

Comment:
As you cleaned up `padding: 0.3em 0.5em;` from the litdev-search :host, I think you can remove this as well.